pkgs/build-support/rust/default.nix

```diff
@@ -181,7 +181,7 @@ stdenv.mkDerivation (args // {
       "CXX_${rust.toRustTarget stdenv.buildPlatform}"="${cxxForBuild}" \
       "CC_${rust.toRustTarget stdenv.hostPlatform}"="${ccForHost}" \
       "CXX_${rust.toRustTarget stdenv.hostPlatform}"="${cxxForHost}" \
-      cargo build \
+      cargo build -j $NIX_BUILD_CORES \
```

Comment:
This is what I cargo-cultured from the other builds:

```suggestion
      cargo build -j ''${enableParallelBuilding:+-j $NIX_BUILD_CORES}
 \
```

Reply:
We should respect `NIX_BUILD_CORES` but will setting `enableParallelBuilding = false` be useful in individual rust packages?

Reply:
I don't know, we can wait for somebody to come up with a valid use case, I guess.

Reply:
we can also make the default be true for `buildRustPackage`, and people can selectively set it to false if it somehow affects the build

Reply:
similar to how cmake will make it default to true

Reply:
It is true already as far as I know.

Reply:
Let's leave it out.